src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -65,45 +80,57 @@
  * 4. Implement the following static method in the concrete class:
  *
  *     public static void main(String[] argv) {
-        new MyConcreteClass().instanceMain(argv);
-    }
-
-
+ *       new MyConcreteClass().instanceMain(argv);
+ *   }
  */
 public abstract class CommandLineProgram {
-    @Option(common=true, optional=true)
+    // Picard CmdLine properties file resource path, placed at this path by the gradle build script
+    private static String PICARD_CMDLINE_PROPERTIES_FILE = "picard/picardCmdLine.properties";
+    private static String PROPERTY_USE_LEGACY_PARSER = "picard.useLegacyParser";
+    private static String PROPERTY_CONVERT_LEGACY_COMMAND_LINE = "picard.convertCommandLine";
+    private static Boolean useLegacyParser;
+
+    @Argument(common=true, optional=true)
     public List<File> TMP_DIR = new ArrayList<File>();
 
-    @Option(doc = "Control verbosity of logging.", common=true)
+    @Argument(doc = "Control verbosity of logging.", common=true)
     public Log.LogLevel VERBOSITY = Log.LogLevel.INFO;
 
-    @Option(doc = "Whether to suppress job-summary info on System.err.", common=true)
+    @Argument(doc = "Whether to suppress job-summary info on System.err.", common=true)
     public Boolean QUIET = false;
 
-    @Option(doc = "Validation stringency for all SAM files read by this program.  Setting stringency to SILENT " +
+    @Argument(doc = "Validation stringency for all SAM files read by this program.  Setting stringency to SILENT " +
             "can improve performance when processing a BAM file in which variable-length data (read, qualities, tags) " +
             "do not otherwise need to be decoded.", common=true)
     public ValidationStringency VALIDATION_STRINGENCY = ValidationStringency.DEFAULT_STRINGENCY;
 
-    @Option(doc = "Compression level for all compressed files created (e.g. BAM and GELI).", common=true)
+    @Argument(doc = "Compression level for all compressed files created (e.g. BAM and GELI).", common=true)
```

Comment:
Another archaic doc that could be fix.  GELI is long gone.  Could you fix this please to say `e.g. BAM and .gz files`?